.github/workflows/react_tests.yml

```diff
@@ -8,8 +8,9 @@ defaults:
 
 jobs:
   build:
-
     runs-on: ubuntu-latest
+    env:
+      JEST_TIMEOUT: 300000
```

Comment:
Do I need to add this? ie were the timeouts only in the nightly pipeline, or did we also see them in the github workflow?

Reply:
@sjha4  I don't remember... :(

Reply:
We never saw the timeouts on github..This was a nightly only issue.